modules/datadict/php/datadict.class.inc

```diff
@@ -76,7 +76,7 @@ class Datadict extends \NDB_Menu_Filter
         $this->searchKeyword = array(
                                 'pt.Name',
                                 'pt.sourceField',
-                                'pto.description',
+                                'pt.description',
```

Comment:
This will make it so that it doesn't search the override (which should have priority over the default parameter_type table.)

Reply:
Thanks David for the help. Now it's set to 'coalesce(pto.description,pt.description)'.